src/posts/what-is-wai.md

```diff
@@ -0,0 +1,108 @@
+
+As described by [W3’s website](https://www.w3.org/Consortium/), “W3C's mission is to lead the Web to its full potential.”
+
+Organizers from all around the world, involved in many different fields, join W3C to develop guidelines that ensure long-term growth for the Web. The Web Accessibility Initiative is one initiative that was launched by W3C.
+
+## What is the Web Accessibility Initiative?
+
+> The power of the Web is in its universality.
+> Access by everyone regardless of disability is an essential aspect.
+> Tim Berners-Lee, W3C Director and inventor of the World Wide Web
+
+W3C launched The Web Accessibility Initiative, (WAI), in 1997, with support from the White House. The Web Accessibility Initiative’s main goal is to make the internet accessible for people with disabilities.
+
+WAI brings people from disability organizations, governments and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities that include auditory, cognitive, neurological, physical, speech and visual disabilities.
+
+In short, WAI develops accessibility standards and offers implementation guidance.
+
+As described by [W3C Website](https://www.w3.org/WAI/about/), WAI “pursues accessibility of the web through these primary activities”:
+
+- ensuring that W3C standards support accessibility
+- developing accessibility guidelines for web content and applications,
+  browsers, and authoring tools
+- developing resources to improve web accessibility evaluation
+  processes and tools
+- supporting education and outreach on web accessibility
+- coordinating with research and development that may impact future  
+  accessibility of the web
+- promoting harmonized international uptake of web accessibility  
+  standards
+
+Two best examples of WAI’s significance in everyday technology life are the Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations that were created by members of WAI.
+
+## Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations:
+
+WAI’s effort created guidelines for developers, designers and people in tech to use every single day. Some of the most used guidelines and applications have been WCAG and WAI-ARIA.
+
+[Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/), or WCAG is a benchmark for web accessibility. WCAG was created with a goal to clearly define technical standards and unifying accessibility standards. WCAG’s standards are categorized under 4 principles.
+
+- Perceivable:
+
+  Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
```

Comment:
The W3C may use the term "user" or "users", but [we use the term "person" or "people"](https://www.a11yproject.com/content-style-guide/#user-or-person-). Can we rephrase instances where "user" and "users" is used to use our preferred terminology instead?